### PR TITLE
chore(profiling): Updates documentation assets for the Profiling product pages

### PR DESCRIPTION
**Pull request by enicolaou-splunk**
…
**Requirements**
- [x] The content follows Splunk guidelines for style and formatting.
- [x] There are no CLI errors when building the docs locally.
- [x] You are contributing original content.

**Describe the change**
- This commit updates the product screenshots and GIF assets related to the AlwaysOn Profiling product pages.
- The `spans-stack-traces.srt` page and the respective references have been removed as the feature has been deprecated.

### CODE REVIEW

File: apm/profiling/get-data-in-profiling.rst [hunk 1]
```diff
@@ -142,4 +142,3 @@ After you set up and enable AlwaysOn Profiling, check that profiling data is com
 4. Select a service, and switch from the CPU view to the Memory view. 
 5. If your service runs in multiple instances, select the instance that you're interested in by selecting the host, container and process ID.
 6. If you've enabled Memory profiling, explore memory metrics. See :ref:`profiling-memory-metrics`.
-7. You can also browse all stack traces coming from your application in the flame graph. See :ref:`flamegraph-howto` for more information about the flame graph. See also :ref:`spans-stack-traces` to learn how to locate and browse call stacks.
```
Comment (jmalin-splunk) [inline apm/profiling/get-data-in-profiling.rst:145]: [enicolaou-splunk](https://github.com/enicolaou-splunk)What is the reason for deleting this line?

File: apm/profiling/intro-profiling.rst [hunk 2]
```diff
@@ -12,7 +12,6 @@ Introduction to AlwaysOn Profiling for Splunk APM
 
    profiling-use-case-library
    get-data-in-profiling
-   spans-stack-traces
```
Comment (jmalin-splunk) [inline apm/profiling/intro-profiling.rst:15]: See my previous comment. Have we removed a feature or was this added by mistake in the previous version of the documentation?

File: apm/profiling/profiling-use-case.rst [hunk 3]
```diff
@@ -97,7 +97,6 @@ Learn more
 --------------------
 
 - For more information on AlwaysOn Profiling and how to start using it, see :ref:`profiling-intro`.
-- For more information on the stack traces in spans, see :ref:`spans-stack-traces`.
```
Comment (jmalin-splunk) [inline apm/profiling/profiling-use-case.rst:100]: See my previous comments about stack traces for spans

**Review by jmalin-splunk — COMMENTED** [review]
This PR removes several images, changes several more, and modifies the documentation for span stack traces. Because there's no Jira ticket associated with the PR, I can't tell if these changes are correct or necessary. Please provide some additional explanation.

File: apm/profiling/get-data-in-profiling.rst (hunk 1 above)
Comment (enicolaou-splunk) [reply apm/profiling/get-data-in-profiling.rst:145]: I'm sorry, I forgot to mention this in the description. Thank you for the question! 
The spans-stack-traces.srt page and the respective references have been removed as the feature has been deprecated.

File: apm/profiling/intro-profiling.rst (hunk 2 above)
Comment (enicolaou-splunk) [reply apm/profiling/intro-profiling.rst:15]: > The spans-stack-traces.srt page and the respective references have been removed as the feature has been deprecated.

**Review by theletterf — APPROVED** [review]

**Pull request merged**